src/AccessibilityInsights.SharedUx/Controls/HierarchyControl.xaml.cs

```diff
@@ -131,6 +131,12 @@ public void SetElement(ElementContext ec, bool expandall = false)
                 }
 
                 UpdateButtonVisibility();
+
+                cmHierarchySettings.Items.Remove(mniShowUncertain);
```

Comment:
realized I'm not sure if this will fail upon rescan - e.g. what if mniShowUncertain is not in the list? A contains check first might work better